src/integrant/core.cljc

```diff
@@ -33,15 +33,17 @@
              (dep/graph)
              config))
 
-(defn read-string
-  "Read a config from a string of edn. Refs may be denotied by tagging keywords
-  with #ref."
-  ([s]
-   (read-string {:eof nil} s))
-  ([opts s]
-   (let [readers (merge {'ref ref} (:readers opts {}))]
-     (edn/read-string (assoc opts :readers readers) s))))
-
+#?
```

Comment:
Rationale for not doing that is essentially every editor i've tried gets the indentation wrong if I do